proto/autoops/service.proto

```diff
@@ -40,7 +40,12 @@ message GetAutoOpsRuleResponse {
 
 message CreateAutoOpsRuleRequest {
   reserved 1;
-  CreateAutoOpsRuleCommand command = 2 [deprecated = true];
+  CreateAutoOpsRuleCommand command = 2 [
+    deprecated = true,
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      description: "deprecated"
+    }
+  ];
   string environment_id = 3 [(google.api.field_behavior) = REQUIRED];
   string feature_id = 4;
   OpsType ops_type = 5;
```

Comment:
These fields are also required when creating an auto operation. We could set it as required.